src/app/core/services/upload-attachments/upload-attachments.service.ts

```diff
@@ -85,13 +89,37 @@ export class UploadAttachmentsService {
     };
 
     // Upload all attachments concurrently
-    const uploadedAttachments = await Promise.all(attachmentsToUpload.map(uploadAttachmentAndHandleErrors));
+    const uploadedAttachments = await Promise.allSettled(attachmentsToUpload.map(uploadAttachmentAndHandleErrors));
 
     if (failedAttachments.length) {
-      throw new UploadAttachmentError(draft.uuid, failedAttachments);
+      // If one of the attachment uploads fails we keep on sending the registration, tho we inform the user with a toast
+      const errorMessage = await firstValueFrom(
+        this.translateService.get(['REGISTRATION.IMAGE_UPLOAD_ERROR'], {
+          failedAttachmentsCount: failedAttachments.length,
+          obsTime: this.formatDate(new Date(draft.registration.DtObsTime)),
+        })
+      );
+      this.createToast(errorMessage);
     }
+    // return attachments that returned a value
+    return [...alreadyUploaded, ...uploadedAttachments.map((p) => p.status === 'fulfilled' && p.value)];
+  }
+
+  private formatDate(dt: Date) {
```

Comment:
OnceAndOnlyOnce! Hvis du trenger nye måter å formatere tidspunkt på, kan du legge funksjonen i date-helper.service, ellers finner du kanskje allerede noe du kan bruke der. Javascript har også en innebygget funksjon for dette som vi bruker i file-logging.service. Vet ikke om den passer her.